Library/Formula/adwaita-icon-theme.rb

```diff
@@ -1,17 +1,7 @@
-require "formula"
-
-class GnomeIconTheme < Formula
+class AdwaitaIconTheme < Formula
```

Comment:
OK. Thanks! Should I submit a new Pull Request, or just add comments on this one?